### ISSUE TITLE
Checksum failed for v0.8.0

### ISSUE THREAD

**Issue by vinhphuctadang** [tryvium-travels/memongo#21]
Hi all contributors,
when I go mod tidy with  `github.com/tryvium-travels/memongo` and `github.com/tryvium-travels/memongo/memongolog`, I faced this issue:

```
github.com/tryvium-travels/memongo/memongolog: github.com/tryvium-travels/memongo@v0.8.0: verifying module: checksum mismatch
        downloaded: h1:+tXjXsGTz8uRpHNkO5OhgZYolaVHh3QJDoBweOTSnIk=
        sum.golang.org: h1:+2h0VpspWyRT6len83UcohbCcKarUBvUgXZj7ezhW2o=
```

, which means checksum for the package is modified, is it expected? Can you guys help check on the go sum server to update checksum?

The workaround method is to turn off the go sum db (`go env -w GOSUMDB=off`), but it's not safe, is it?

**Comment by saniales** [comment]
Probably you were facing the error because the v0.8.0 had a tag but not a release, I released it now, can you tell if you still have the error?

**Comment by vinhphuctadang** [comment]
I still see the issue unfortunately, maybe it's from `sum.golang.org`?

<img width="600" alt="image" src="https://user-images.githubusercontent.com/30641530/210029643-1f79a1d6-080d-4809-aa28-917c2651fb99.png">

**Comment by saniales** [comment]
we will release a new version soon, hopefully with the version bump the problem will be solved

**Comment by saniales** [comment]
solved with new version v0.9.0
